index.html

```diff
@@ -360,7 +360,7 @@ <h2 class="contact">Contact Me</h2>
             to help with it
           </p>
           <form
-            action="https://formspree.io/f/xvolqjkn"
+            action="#"
```

Comment:
Nice job 👍 in checking the email validation. But currently, when the user submits a valid form the action it's returning a 403 error 👇 . 

<img width="453" alt="image" src="https://user-images.githubusercontent.com/34578631/162176431-086fdb67-0b7e-4657-add6-ed3ac21cf619.png">

Kindly, update the form action to submit the form properly. You can follow this [link](https://formspree.io/html) on how to set the form action.